doc/sphinx/language/core/inductive.rst

```diff
@@ -8,14 +8,15 @@ Inductive types
 
 .. cmd:: Inductive @inductive_definition {* with @inductive_definition }
 
-   .. insertprodn inductive_definition cumul_ident_decl
+   .. insertprodn inductive_definition cumul_univ_decl
 
    .. prodn::
       inductive_definition ::= {? > } @cumul_ident_decl {* @binder } {? %| {* @binder } } {? : @type } {? := {? @constructors_or_record } } {? @decl_notations }
       constructors_or_record ::= {? %| } {+| @constructor }
       | {? @ident } %{ {*; @record_field } {? ; } %}
       constructor ::= @ident {* @binder } {? @of_type }
       cumul_ident_decl ::= @ident {? @cumul_univ_decl }
```

Comment:
Since it is used only in one place, I suggest to splice `cumul_ident_decl`. IMHO, this will make `inductive_definition` more readable.